tests/test_helpers.py

```diff
@@ -93,3 +97,72 @@ def test_flatten_html_formats_unordered_lists(
     input: str, expected: str
 ) -> None:
     assert helpers.flatten_html_string(input) == expected
+
+
+if sys.platform.startswith("win"):
+    import msvcrt
+
```

Comment:
To spare the indentation one can use `@pytest.mark.skipif(sys.platform.startswith("linux"), reason="does not run on linux")`. The other test could use `@pytest.mark.skipif(sys.platform.startswith("win"), reason="does not run on windows")`. The import would happen in the body of the test function.